src/include/connection.h

```diff
@@ -67,6 +67,17 @@ struct __wt_background_compact_stat {
     TAILQ_ENTRY(__wt_background_compact_stat) hashq;
 };
 
+/*
+ * WT_BACKGROUND_COMPACT_EXCLUDE_ENTRY --
+ *	An entry indicating this URI should be excluded from background compaction.
+ */
+struct __wt_background_compact_exclude_entry {
+    const char *name; /* File name */
```

Comment:
Generally speaking, when do we use "file name" as opposed to "file uri" in the code?

Reply:
I believe they are used interchangeably. I am happy to change it to `uri` if you would prefer.

Reply:
Since we have stripped "table:" from the URI before storing in entry->name (correct me if that is not true), these are no longer uri, these are filenames now.

Reply:
Should I change the description then @sulabhM?
```
An entry indicating this URI should be excluded from background compaction.
```
Should we change `URI` to something else?

Reply:
Ideally yes, there are a couple of other places we say "URI" - we should call file names.

Reply:
Thanks, done in 37786d25fe621313ecf11112bcd623c864afdfd4.

Reply:
Thanks, that makes sense.